frameworks/Go/go-mongodb/bash_profile.sh

```diff
@@ -0,0 +1,4 @@
+# Set the root of our go installation
+export GOROOT=${IROOT}/go
```

Comment:
Let's move these lines into the top of the `install.sh` file - see #1259 for the reason